WORKSPACE

```diff
@@ -163,6 +164,14 @@ load("@aspect_rules_js//npm:repositories.bzl", "npm_translate_lock")
 
 npm_translate_lock(
     name = "npm2",
+    custom_postinstalls = {
+        "@angular/animations": "node ../../@nginfra/angular-linking/index.mjs",
```

Comment:
I don't understand where this is happening, or how?

It seems odd that this is going above the workspace. in the directory structure?

Reply:
It's a `rules_js` detail. The postinstall executes inside the package. So going up twice, gives you access to the package dependencies.

Reply:
Where is @nginfra coming from?

Reply:
https://github.com/devversion/angular-linking